wasmapi/go/containers.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2024 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import (
+	"fmt"
+)
+
+//go:wasmimport env getContainers
+func getContainers() uint32
```

Comment:
Somewhere it should be documented that this function returns the containers on the CURRENT host.

In a K8s scenario the WASM developer might think it could return all containers from the whole Cluster